biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -357,6 +360,47 @@ public void analyze() throws Exception {
 						uses.transpose().get(Descriptors.DEFAULT_PACKAGE));
 			}
 
+			// Check for use of the deprecated bnd @Export annotation
+
+			TypeRef bndAnnotation = descriptors.getTypeRefFromFQN(aQute.bnd.annotation.Export.class.getName());
```

Comment:
Rather than this stream forEach, put the warning for using the Bnd Export annotation in the case statement below.